src/createXcmTypes/util/assetIdsContainsRelayAsset.spec.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+
+import { Registry } from '../../registry';
+import { assetIdsContainRelayAsset } from './assetIdsContainsRelayAsset';
+
+describe('assetIdsContainsRelayAsset', () => {
+	// it('Should return true when assetIds contains relay chain symbol', () => {
```

Comment:
This test is commented out, just as a note.

Reply:
great catch. uncommenting